docs/quickstart.mdx

```diff
@@ -94,7 +101,9 @@ Attached to [registry] localhost:5000/hello-artifact@sha256:327db68f73d0ed53d528
 Digest: sha256:bcdd6799fed0fca0eaedfc1c642f3d1dd7b8e78b43986a89935d6fe217a09cee
 ```
 
-### Step 3.3: View referrers 
+### Step 3: View referrers 
+
+We can use [`oras discover`](./commands/oras_discover.mdx) to view the referrers.
 
 ```
 oras discover localhost:5000/hello-artifact:v1
```

Comment:
Could you try `oras discover localhost:5000/hello-artifact:v1 -o tree` and copy the result into the result below? It will show the referrer's tree view.